addon/adapters/json-api.js

```diff
@@ -99,7 +100,7 @@ var JSONAPIAdapter = RESTAdapter.extend({
     @return {Promise} promise
   */
   findMany(store, type, ids, snapshots) {
-    if (isEnabled('ds-improved-ajax')) {
+    if (isEnabled('ds-improved-ajax') && !this._hasCustomizedAjax()) {
```

Comment:
Should be allowed, as that was released as 0.2.1 and we are depending on `^0.2.0` ([see here](https://github.com/emberjs/data/blob/master/package.json#L27)).